src/i18n/locales/en-US.json

```diff
@@ -22,13 +22,13 @@
   "components.delegationsummary.epochProgress.title": "Epoch progress",
   "components.delegationsummary.failedwalletupgrademodal.title": "Heads up!",
   "components.delegationsummary.failedwalletupgrademodal.explanation1": "Some users experienced problems while upgrading their wallets in the Shelley testnet.",
-  "components.delegationsummary.failedwalletupgrademodal.explanation2": "If you observed an unexpected zero balance after having upgraded your wallet, we recommend you to restore your wallet once again. We apologize for any inconvinience this may have caused.",
+  "components.delegationsummary.failedwalletupgrademodal.explanation2": "If you observed an unexpected zero balance after having upgraded your wallet, we recommend you to restore your wallet once again. We apologize for any inconvenience this may have caused.",
   "components.delegationsummary.failedwalletupgrademodal.okButton": "OK",
   "components.delegationsummary.notDelegatedInfo.firstLine": "You have not delegated your ADA yet.",
-  "components.delegationsummary.notDelegatedInfo.secondLine": "Go to Staking center to choose which stake pool you want to delegate in. Note, you may delegate only to one stake pool in this Tesnnet.",
```

Comment:
as far as I know, this restriction will apply to Haskell Shelley for the foreseeable future also